### PR TITLE
media queries

### PR DESCRIPTION
**Pull request by mbenotti**
added code, but it doesn't seem to be working - what have I missed? Please advise - Thanks!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -38,9 +38,18 @@
...
+.footer {
   height: 60px;
   padding-top: 10px;
   background-color: #000000; /*black*/
   color: white;
   }
+  @media screen and (min-width: 768px)
+    header {
+     background-color: #ffa500; /*orange*/
+    }
```
Comment (ayojackay) [inline css/styles.css:51]: I caught what is missing. Look at the `@media screen and (min-width: 768px)` . Below, you mention the `header`, BUT if you look closely, its like a stand alone. How does the `header` know it is suppose to live *INSIDE* the media query? I'll give you a hint `{}` .

File: css/styles.css [hunk 2]
```diff
@@ -38,9 +38,18 @@
...
+  @media screen and (min-width 875px)
```
Comment (ayojackay) [inline css/styles.css]: same issue as above. When you catch it, add it to both media queries.

File: css/styles.css (hunk 1 above)
Comment (ayojackay) [reply css/styles.css:51]: If you can see my comment now, please let me know. I redid it again. (BUT IGNORE IT you already fixed the issue). I just wanted you to see what the initial comment was. Thanks.

Comment (mbenotti) [reply css/styles.css:51]: I can see your comment now - I wonder why it didn't work before.

**Pull request merged**